Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -4928,7 +4928,20 @@ def start_site_scan_command(client: Client, site_id: str | None = None, site_nam
 
     else:
         assets = client.get_site_assets(site.id)
-        hosts_list = [asset["ip"] for asset in assets]
+
```

Comment:
please add test to this use case :)

Reply:
Done in a838794102587104526ea99e959186702fb711b9